integration/misc.py

```diff
@@ -0,0 +1,88 @@
+import time
+import weaviate
+
+from integration.integration_util import testmethod
+
+
+schema = {
+    "classes": [
+        {
+            "class": "ClassA",
+            "properties": [
+                {
+                    "dataType": [
+                        "string"
+                    ],
+                    "name": "stringProp"
+                },
+                {
+                    "dataType": [
+                        "int"
+                    ],
+                    "name": "intProp"
+                }
+            ]
+        }
+    ]
+}
+
+class TestMisc:
+    def __init__(self, client: weaviate.Client):
+        client.schema.delete_all()
+        self.client = client
+        self.git_hash = "2147e86"
+        self.server_version = "1.15.4"
```

Comment:
Do we need to change this every time we update the image?

Reply:
Yes, this is in parity with the other 3 clients

Reply:
Is there a better way to do this? Autodetect it somehow? I think this could become really annoying while developing

Reply:
It only needs to be changed each time the weaviate image is updated in the docker compose file. But I agree that a better way should be sought out